### PR TITLE
Pream

### PR DESCRIPTION
**Pull request by preamza02**
ทำตาม https://docs.google.com/document/d/1HLeDNsQpmkt7CEcFS9kdMgCxUQFsOB_N4vmu-Lu20PE/edit
แก้ logo หน้า Login
ตั้งเวลาเปลี่ยน text end fake date 30 ก.ย.

### CODE REVIEW

File: src/core/constants/timeline.constant.tsx [hunk 1]
```diff
@@ -5,61 +5,62 @@ export interface TimelineModel {
...
   },
   {
     duration: {
-      start: new Date(2565, 0, 31),
-      end: new Date(2565, 1, 6)
+      start: new Date(2565, 9, 29),
+      end: new Date(2565, 9, 29)
     },
     title: "สัมภาษณ์รอบต่างจังหวัด"
   },
```
Comment (Nitiwat-owen) [inline src/core/constants/timeline.constant.tsx]: Unfortunately, the interview day has changed to 30 October 2022.

**Review by Nitiwat-owen — CHANGES_REQUESTED** [review]
Could you make the text be center or reduce the height of the background picture.
image

**Review by Nitiwat-owen — CHANGES_REQUESTED** [review]
Reduce the gap between Facebook and Instagram embedded
Screen Shot 2565-09-02 at 18 59 49

**Review by Nitiwat-owen — CHANGES_REQUESTED** [review]
Could you style the text be inline.
Screen Shot 2565-09-02 at 19 02 46

**Review by Nitiwat-owen — CHANGES_REQUESTED** [review]
As you can see that there are some errors because of missing the key property of the mapping component, so I would you to add the key props in that missing component.
Screen Shot 2565-09-02 at 19 09 14

File: src/core/providers/announce.provider.tsx [hunk 2]
```diff
@@ -15,21 +15,21 @@ const SCHEDULE = [
...
-    end: new Date("November 15, 2021 10:00:00 GMT+07:00")
+    end: new Date("September 12, 2022 10:00:00 GMT+07:00")
   },
   {
     state: ApplicationStatus.APPLICABLE,
-    start: new Date("November 15, 2021 10:00:00 GMT+07:00"),
-    end: new Date("December 16, 2021 00:30:00 GMT+07:00")
+    start: new Date("September 12, 2022 10:00:00 GMT+07:00"),
+    end: new Date("October 8, 2022 00:30:00 GMT+07:00")
   },
```
Comment (Nitiwat-owen) [inline src/core/providers/announce.provider.tsx]: We will let the student apply the form at 6pm, so please edit the time openning.

File: src/core/constants/timeline.constant.tsx [hunk 3]
```diff
@@ -5,61 +5,62 @@ export interface TimelineModel {
...
   {
     duration: {
-      start: new Date(2565, 5, 2),
-      end: new Date(2565, 5, 5)
+      start: new Date(2566, 0, 26),
+      end: new Date(2566, 0, 29)
     },
     title: "ค่ายลานเกียร์ครั้งที่ 22"
   }
```
Comment (Nitiwat-owen) [inline src/core/constants/timeline.constant.tsx]: Also, the camp has changed to 5 - 8 Jan 23. I think you have to change the vaccine qualification too.  Please change that the applicant must get the vaccine before 22 Dec 2022.

**Comment by fromsukong** [comment]
new commit : 
fix from commend
fix logo size in sidebar
fix gap below Instagram embed

**Comment by fromsukong** [comment]
> Reduce the gap between Facebook and Instagram embedded Screen Shot 2565-09-02 at 18 59 49

i reduce the gap from 6 px to 4 px  
image

**Review by Nitiwat-owen — CHANGES_REQUESTED** [review]

File: src/core/constants/contact.constant.tsx [hunk 4]
```diff
@@ -4,10 +4,10 @@ export interface ContactModel {
...
 const contactConstant: Array<ContactModel> = [
-  { name: "พี่รีฟ", tel: "092-393-5154" },
-  { name: "พี่มิก", tel: "080-296-0233" },
-  { name: "พี่จินนี่", tel: "087-391-4644" },
-  { name: "พี่ดราก้อน", tel: "081-820-9505" }
+  { name: "พี่ตีตี้", tel: "092-508-8667" },
+  { name: "พี่บุ๊ค", tel: "095-595-1000" },
+  { name: "พี่เจิ้ง", tel: "091-950-3579" },
+  { name: "พี่เฟรนด์", tel: "083-609-1461" }
 ]
```
Comment (Nitiwat-owen) [inline src/core/constants/contact.constant.tsx]: Boards want to change the contact information to be id line.
พี่ตีตี้  line id : tity1506
พี่บุ๊ค  line id : book11653
พี่เจิ้ง  line id : pasana_jerng
พี่เฟรน line id : friendlypp
which I think you have to change the icon also.

**Review by Nitiwat-owen — CHANGES_REQUESTED** [review]

File: src/core/constants/contact.constant.tsx (hunk 4 above)
Comment (fromsukong) [reply src/core/constants/contact.constant.tsx]: i have changed it

**Review by Nitiwat-owen — CHANGES_REQUESTED** [review]
Could you please decrease the size of file bg6.jpg to be as same as the others (width : 1567px, height : 1045px).

**Comment by fromsukong** [comment]
> 

hi i have changed it

**Review by Nitiwat-owen — APPROVED** [review]

**Pull request merged**